scaffolds/scala/src/test/scala/euranova/AppleTest.scala

```diff
@@ -0,0 +1,17 @@
+package euranova
+
+import org.scalatest.FunSuite
+
+class MyTest extends FunSuite{
+
+	test("prop's val is OK") {
```

Comment:
You are claiming to test that the value is OK, but you are testing in fact that it's not not OK which isn't entirely the same thing.